{
  "id": "CVE-2020-4388",
  "assigner": "psirt@us.ibm.com",
  "Published": "2020-10-12T12:15:12.543Z",
  "Modified": "2020-10-14T11:25:33.233Z",
  "last-modified": "2020-10-14T11:25:33.233Z",
  "summary": "IBM Cognos Analytics 11.0 and 11.1 could be vulnerable to a denial of service attack by failing to catch exceptions in a servlet also exposing debug information could also be used in future attacks. IBM X-Force ID: 179270.",
  "access": {
    "authentication": "NONE",
    "complexity": "LOW",
    "vector": "NETWORK"
  },
  "impact": {
    "availability": "PARTIAL",
    "confidentiality": "PARTIAL",
    "integrity": "NONE"
  },
  "cvss": 6.4,
  "exploitabilityScore": 10.0,
  "impactScore": 4.9,
  "cvss-vector": "AV:N/AC:L/Au:N/C:P/I:N/A:P",
  "references": [
    "https://exchange.xforce.ibmcloud.com/vulnerabilities/179270",
    "https://www.ibm.com/support/pages/node/6346922"
  ],
  "vulnerable_configuration": [
    "cpe:2.3:a:ibm:cognos_analytics:11.0.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.0.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.1.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.2:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.2.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.3:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.3.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.4:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.4.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.5:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.5.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.6:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.6.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.7:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.7.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.8:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.9:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.10:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.10.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.11:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.11.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.12:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.12.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.2:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.3:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.4:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.5:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.6:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:-:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack1:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack2:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack3:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack4:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack5:*:*:*:*:*:*"
  ],
  "vulnerable_product": [
    "cpe:2.3:a:ibm:cognos_analytics:11.0.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.0.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.1.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.2:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.2.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.3:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.3.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.4:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.4.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.5:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.5.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.6:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.6.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.7:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.7.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.8:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.9:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.10:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.10.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.11:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.11.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.12:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.0.12.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.0:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.1:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.2:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.3:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.4:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.5:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.6:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:*:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:-:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack1:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack2:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack3:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack4:*:*:*:*:*:*",
    "cpe:2.3:a:ibm:cognos_analytics:11.1.7:fixpack5:*:*:*:*:*:*"
  ],
  "vendors": [
    "ibm"
  ],
  "products": [
    "cognos_analytics"
  ],
  "vulnerable_product_stems": [
    "cpe:2.3:a:ibm:cognos_analytics"
  ],
  "vulnerable_configuration_stems": [
    "cpe:2.3:a:ibm:cognos_analytics"
  ],
  "cwe": "CWE-755",
  "vulnerable_configuration_cpe_2_2": []
}